tests/scripts/thread-cert/pcap.py

```diff
@@ -43,7 +43,6 @@ class PcapCodec(object):
     def __init__(self, filename):
         self._pcap_file = open('%s.pcap' % filename, 'wb')
         self._pcap_file.write(self.encode_header())
-        self._epoch = time.time()
```

Comment:
Explain: make Real Time sniffer use the real time (Unix Time) so that the pcap can be merged with Backbone pcap properly.